### DISCUSSION TITLE
Unrelated data

### DISCUSSION THREAD

**Discussion by jcphlux** [msiemens/tinydb#474] [Q&A] [answered]
Is it best to store unrelated data in separate instances/files of TinyDB?

**Comment by msiemens** [comment, marked as answer]
That depends. Personally I'd most probably use separate tables for unrelated data because otherwise I'd have to juggle multiple TinyDB instances in parallel. But splitting data into separate files could (at least in theory) improve performance for large datasets as writing one dataset doesn't require also writing the other dataset. But if performance is such a concern that things like this make a big difference you're probably reaching the limits of TinyDB 🙂
